scripts/githooks/check-file-format.sh

```diff
@@ -70,13 +67,42 @@ function main() {
       ;;
   esac
 
+  if command -v editorconfig > /dev/null 2>&1 && ! is-arg-true "${FORCE_USE_DOCKER:-false}"; then
+    filter="$filter" dry_run_opt="${dry_run_opt:-}" cli-run-editorconfig
+  else
+    filter="$filter" dry_run_opt="${dry_run_opt:-}" docker-run-editorconfig
+  fi
+}
+
+# Run editorconfig natively.
+# Arguments (provided as environment variables):
+#   dry_run_opt=[dry run option]
+#   filter=[filter for files to check]
```

Comment:
I think these comments could be removed if the things they are describing are more clear
cli-run-editorconfig() -> native-run-editorconfig() _(or something similar)_

Reply:
This is the header of the function. I suggest we keep it as it communicates the dependency on the variables needed to be set for the function to run correctly. However, I'll rename this function (and others) for clarity as recommended. Thanks.

e.g.
- `run-editorconfig-natively`
- `run-editorconfig-in-docker`

Reply:
I removed the headers across the codebase where there are no arguments to be passed, and the name of the function clearly expresses its intent.